blog/2022-02-10-automating-dhis2-integration-tests-in-junit-5.md

```diff
@@ -1,7 +1,7 @@
 ---
 slug: 2022/02/automating-dhis2-integration-tests-in-junit-5
 title: Automating tests for DHIS2 integrations with JUnit 5
-author: claudemamo
```

Comment:
You should instead add yourself as an author [here](https://github.com/dhis2/developer-portal/blob/fc9a5cea7a5809c3027ae0fa850581f64155a1cf/blog/authors.yml) and include the key to that collection as the `author` field